samples/explore_workbook.py

```diff
@@ -74,6 +74,10 @@ def main():
         if all_workbooks:
             # Pick one workbook from the list
             sample_workbook = all_workbooks[0]
+            sample_workbook.name = "Name me something cooler"
+            sample_workbook.description = "That doesn't work"
```

Comment:
To confirm, updating the description doesn't work?

Reply:
Yes - it's an open bug, hopefully fixed soon.